textalyze.js

```diff
@@ -17,7 +19,87 @@ function itemCounts(array) {
   // Your code here.
   // Run 'npm test' to see what tests need to pass.
 
+  array.forEach((value) => {
+    const sum = counts.get(value) ? counts.get(value) + 1 : 1;
```

Comment:
Since we know `counts` is a `Map` whose values are either a `number` or `undefined`, we could do:

```javascript
const currentCount = counts.get(value) || 0;
counts.set(value, currentCount + 1);
```

Reply:
Yes quite better